Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10
+
```

Comment:
Sure, for the final build you would not want dev dependencies, you can use multi stage builds for this.
In this model the first stage would be to build the development image and generate the files in the post build step.
The final optional stage in the build would be to create a production image and copy the css files across from the previous image.
Further details here https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Multi-stage builds are great but I think, for our current purposes, it would be an unnecessary complication. We can simply run `npm ci` again with the correct `NODE_ENV` to remove dev dependencies at the end of the build. Would you agree with this @markysoft?

Reply:
On further inspection, I've found out that since the last time I looked at multi-stage builds, it has become possible to specify an intermediate build in `docker-compose` or a `docker run` command. That adds a lot of value as we can now build both development and production containers from the same Dockerfile without any awkward ENV checks in the build process.